Change task dependency to mustRunAfter

enpuyou opened the pull request:
This changes the task dependency from `dependOn` to `mustRunAfter` due to the problem that it doesn't support the use of tag `--continue`, which continues the execution of tasks even if the previous one failed. In this case, to let `report` run even `grade` is not passed. The correct command `gradle --continue grade report`(the order of `grade` and `report` does not matter, `--continue` can also be put at the end of the command like this: `gradle grade report --continue`) is also updated in the `README`

This PR is a bugfix that fixes #68

enpuyou committed:
Change to mustRunAfter due to dependOn does not work with tag --continue

enpuyou committed:
update README

Michionlion commented:
Do we handle the situation `gradle report` gracefully? Is there a good error message? If so, I'll merge this and do an emergency update to the Plugin repository so that `v0.5.0` uses this bugfix.

enpuyou committed:
Add exception handling

Michionlion commented:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/main/java/org/gatorgradle/task/GatorGradleReportTask.java  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/GatorEducator/gatorgradle/pullRequest?prid=4751547&bid=15850412)

enpuyou committed:
Adding font color to the message

enpuyou committed:
change exception message to be consistent with

enpuyou committed:
Fix checkstyle import error

enpuyou committed:
fix line too long

Michionlion commented:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/main/java/org/gatorgradle/task/GatorGradleReportTask.java  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/GatorEducator/gatorgradle/pullRequest?prid=4751547&bid=15850412)

Michionlion committed:
Update gradle.properties

This changes the Gradle artifact version to 0.7.1 -- which will end up lining up with pr-tag-release's numbering. In the future an infrastructure update will bring this inline with the tagged versioning

Michionlion reviewed (approved):
This looks good! I just updated the manually set version, as I forgot about the pr-tag-release automation we added a while back. This will bring things together to be correct, although we will skip `v0.6.0`.

Michionlion commented:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/main/java/org/gatorgradle/task/GatorGradleReportTask.java  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/GatorEducator/gatorgradle/pullRequest?prid=4751547&bid=15850412)
